receiver/cloudfoundryreceiver/receiver_test.go

```diff
@@ -14,4 +14,52 @@
 
 package cloudfoundryreceiver
 
-// TODO: integration test to be added in next PR
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/component/componenttest"
+	"go.opentelemetry.io/collector/consumer/consumertest"
+)
+
+// Test to make sure a new receiver can be created properly, started and shutdown with the default config
+func TestDefaultValidReceiver(t *testing.T) {
```

Comment:
This should just be another test in this table test: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/013177e592b724dc217dc9ad958242db76158f4e/internal/components/receivers_test.go#L43